### ISSUE TITLE
Would Whisper require M1 GPU support?

### ISSUE THREAD

**Issue by scottleibrand** [danielgross/teleprompter#2]
To use Whisper for an app like this, I think we’d first want it to add M1 GPU support, as running even the tiny model on CPU is barely above 1x speed for transcription. 

https://github.com/openai/whisper/pull/382 isn’t yet merged, and it’s unclear to me what is required to get it working.

Thoughts?

**Comment by scottleibrand** [comment]
Maybe not, in light of https://github.com/danielgross/teleprompter/pull/1 ?

**Comment by pavanagrawal123** [comment]
@scottleibrand, Yeah, I believe with the smaller models the `whisper.cpp` implementation is good at running at real-time speed. I ran it on CPU only and got pretty good performance, I believe this is very doable without GPU support.
